lib/slim_fast/parser.ex

```diff
@@ -76,6 +77,15 @@ defmodule SlimFast.Parser do
   defp inline_children("=" <> content), do: [parse_eex(content, true)]
   defp inline_children(input), do: [String.strip(input, ?") |> parse_eex_string]
 
+  defp parse_comment("!" <> comment), do: {:html_comment, children: [String.strip(comment)]}
+  defp parse_comment("[" <> comment) do
+    [h|[t|_]] = comment |> String.split("]", parts: 2)
+    conditions = String.strip(h)
+    children = t |> String.strip |> inline_children
+    {:ie_comment, content: conditions, children: children}
+  end
+  defp parse_comment(comment), do: {:comment, []}
```

Comment:
Is there any possible needs in this node, instead of just `""`? With empty string we do not need special self-closing tag for comment.